extensions/LICENSE

```diff
@@ -0,0 +1,27 @@
+Mypy extensions are licensed under the terms of the MIT license, reproduced below.
+
+= = = = =
+
+The MIT License
+
+Copyright (c) 2015-2016 Jukka Lehtosalo and contributors
```

Comment:
Shouldn't this be updated to year 2017?

Reply:
AIUI, from what I have read updating it is not needed, but I will go ahead and make that change if you want.

Reply:
I am not an expert myself, but this is what I have seen typically.

Reply:
Updated.